tests/integration/workflows/java_gradle/test_java_gradle.py

```diff
@@ -2,34 +2,45 @@
 import shutil
 import tempfile
 
-from zipfile import ZipFile
 from unittest import TestCase
+from pathlib import Path
 
 from aws_lambda_builders.builder import LambdaBuilder
 from aws_lambda_builders.exceptions import WorkflowFailedError
+from tests.integration.workflows.common_test_utils import does_folder_contain_all_files, does_folder_contain_file
+
+
+p = os.path.join
```

Comment:
I see some inconsistency in the use of this function here, some used p yet some used os.path.join, can we make them consistent here?